README.adoc

```diff
@@ -143,8 +143,16 @@ ruby_bundle(
 # You can specify more than one bundle in the WORKSPACE file
 ruby_bundle(
     name = "bundle_app_shopping",
-    gemfile = "//apps/shopping:Gemfile",
-    gemfile_lock = "//apps/shopping:Gemfile.lock",
+    gemfile = "//:apps/shopping/Gemfile",
+    gemfile_lock = "//:apps/shopping/Gemfile.lock",
```

Comment:
Can you explain why the : is needed?

Reply:
I think the original documentation is not working because the colon should preceed directory name. At least that's what I had to do in Selenium where all Ruby code lives in `rb` directory (https://github.com/SeleniumHQ/selenium/blob/trunk/WORKSPACE#L286):

```bazel
ruby_bundle(
    name = "bundle",
    srcs = [
        "//:rb/lib/selenium/devtools/version.rb",
        "//:rb/lib/selenium/webdriver/version.rb",
        "//:rb/selenium-devtools.gemspec",
        "//:rb/selenium-webdriver.gemspec",
    ],
    gemfile = "//:rb/Gemfile",
)
```